src/loopback.c

```diff
@@ -105,16 +105,16 @@ int loopback_write_open (hashcat_ctx_t *hashcat_ctx)
 
   hc_asprintf (&loopback_ctx->filename, "%s/%s.%d_%u", induct_ctx->root_directory, LOOPBACK_FILE, (int) now, random_num);
 
-  FILE *fp = fopen (loopback_ctx->filename, "ab");
+  HCFILE fp;
 
-  if (fp == NULL)
+  if (hc_fopen (&fp, loopback_ctx->filename, "ab", HCFILE_FORMAT_PLAIN) == false)
```

Comment:
Just a reminder to remove third parameter here as well.

Reply:
k